src/en/kunmangato/src/eu/kanade/tachiyomi/extension/en/kunmangato/KunMangaTo.kt

```diff
@@ -0,0 +1,267 @@
+package eu.kanade.tachiyomi.extension.en.kunmangato
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.FormBody
+import okhttp3.Headers
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import uy.kohesive.injekt.injectLazy
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Locale
+import kotlin.concurrent.thread
+
+class KunMangaTo : ParsedHttpSource() {
+    override val name = "KunMangaTo"
+    override val baseUrl = "https://kunmanga.to"
+    override val lang = "en"
+    override val supportsLatest = true
+
+    private val json: Json by injectLazy()
+
+    private val dateFormat by lazy {
+        SimpleDateFormat("MMM d, yyyy", Locale.ENGLISH)
+    }
+
+    override fun popularMangaRequest(page: Int): Request {
+        return GET(baseUrl)
+    }
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        val document = response.asJsoup()
+        val mangas = document.select(popularMangaSelector()).map { element ->
+            popularMangaFromElement(element)
+        }.associateBy({ it.url }, { it }).values.toList()
+        return MangasPage(mangas, false)
+    }
+
+    override fun popularMangaSelector(): String {
+        return ".sidebar-box-popular article"
+    }
```

Comment:
Can be write like this too
```suggestion
    override fun popularMangaSelector(): String = ".sidebar-box-popular article"
```

Reply:
Fixed